app/services/audio.js

```diff
@@ -501,19 +523,19 @@ export default Service.extend({
       this._trackPlayerEventForNpr({
         category: 'Engagement',
         action: 'Stream_Pause',
-        label: `Streaming_${get(audio, 'name')}`
+        label: `Streaming_${get(storyOrStream, 'name')}`
       });
     } else if (type === 'ondemand') {
       this._trackPlayerEventForNpr({
         category: 'Engagement',
         action: 'On_demand_audio_pause',
-        label: get(audio, 'audio')
+        label: get(storyOrStream, 'audio')
       });
     }
 
-    if (audio && get(audio, 'audioType') !== 'stream') {
+    if (storyOrStream) {
       // we're not set up to handle pause listen actions from streams atm
-      this.sendPauseListenAction(audio.id);
+      this.sendListenAction(storyOrStream, 'pause');
```

Comment:
Ah yeah, that comment can be updated, I think. At worst we'll get a 404 when this passes through to the legacy listen action endpoint (see _legacySend in data pipeline service).

We can remove this comment post-deploy. I can also update publisher to quietly fail on stream pause listen actions in the interim, but I think we'll be getting rid of all that from publisher fairly soon.